platform/ios/display_server_ios.mm

```diff
@@ -108,6 +88,18 @@
 	}
 #endif
 
+#if defined(GLES3_ENABLED)
+	if (rendering_driver == "opengl3") {
+		CALayer *layer = [AppDelegate.viewController.godotView initializeRenderingForDriver:@"opengl3"];
+
+		if (!layer) {
+			ERR_FAIL_MSG("Failed to create iOS rendering layer.");
```

Comment:
Should maybe make the Vulkan and OpenGL3 respective failure messages more explicit so that users can find in the logs which renderer was attempted and failed?

Reply:
There will be an error message from the `initializeRenderingForDriver` with more details, but I have added renderer name to these messages as well.